starter-project-web-vue2/components/aymen/BlogList.vue

```diff
@@ -1,6 +1,16 @@
 <template>
   <v-container>
-    <h2>Latest Blogs</h2>
+    <v-toolbar flat app>
+      <v-toolbar-title class="text-uppercase grey--text">
+        <span class="font-weight-light">Latest</span>
+        <span>Blogs</span>
+      </v-toolbar-title>
+      <v-spacer></v-spacer>
```

Comment:
also good to hide the signout button, if a user is not logged in. or change it to sign in